test/conftest.py

```diff
@@ -45,5 +45,5 @@ def pytest_runtest_makereport(item, call):
             )
         except FileNotFoundError:
             return
-        if proc.returncode == 0:
-            rep.sections.append(("Recent API server logs", proc.stdout))
+        # if proc.returncode == 0:
+        #     rep.sections.append(("Recent API server logs", proc.stdout))
```

Comment:
Can this be put back please? 

Have it check an environment variable (via Python's `os.environ`) to decide whether to include this section. I'm ok with the default being off. We can turn it on for travis runs.